### ISSUE TITLE
Prevent system-update-cleanup from deleting flag newer than boot time

### ISSUE THREAD

**Issue by charles-dyfis-net** [systemd/systemd#15883]
Sometimes an update requires more work to be done after a reboot. In such scenarios, it's reasonable for a script invoked from `systemd-update.service` to want to create a new system-update flag to control the *next* boot.

Presently, `system-update-cleanup.service` deletes any such flag.

Consider replacing `/bin/rm -fv /system-update` with `/usr/bin/find /system-update -mindepth 0 -maxdepth 0 ! -newermm /proc/1 -delete`

**Comment by poettering** [comment]
makes sense. But we probably should do that without puling in find. maybe have a separate tool for that
